### ISSUE TITLE
Allow line breaks in jvmOptions

### ISSUE THREAD

**Issue by bmarwell** [jreleaser/jreleaser#1699]
**Is your feature request related to a problem? Please describe.**

Part of the feature assemble javaArchive:

`assemble -> javaArchive -> $archiveName -> java -> jvmOptions`

Currently, line breaks are preserved in `jvmOptions -> universal`.
Example:

```xml
                      <universal>
                        -Djdk.includeInExceptions=hostInfo
                        -Dfile.encoding=UTF-8
                        -Xms64m
                        -Xsoftmx=128m
                        -Xmx256m
                        -Xtune:virtualized
                        -Xcompressedrefs
                        -Xshareclasses:name=myapp
                        -Xshareclasses:nonfatal
                        -Xscmx=64m
                        -XX:SharedCacheHardLimit=128m
                        -XX:+IgnoreUnrecognizedVMOptions
                        -XX:+ExitOnOutOfMemoryError
                        -XX:ActiveProcessorCount=4
                      </universal>
```

Will result in:

```
# universal JVM options
JAVA_OPTS="$JAVA_OPTS -Djdk.includeInExceptions=hostInfo
                        -Dfile.encoding=UTF-8
                        -Xms64m
                        -Xsoftmx=128m
                        -Xmx256m
                        -Xtune:virtualized
                        -Xcompressedrefs
                        -Xshareclasses:name=myapp
                        -Xshareclasses:nonfatal
                        -Xscmx=64m
                        -XX:SharedCacheHardLimit=128m
                        -XX:+IgnoreUnrecognizedVMOptions
                        -XX:+ExitOnOutOfMemoryError
                        -XX:ActiveProcessorCount=4"
```

**Describe the solution you'd like**

I think that linebreaks are really not important for JAVA_OPTS.
Just replace them with banks.

**Describe alternatives you've considered**

Putting everything into one line, but that will usually exceed the maximum line length of my project.

**Additional context**

./.

**Comment by aalmiray** [comment]
Either replace line breaks with space _or_ escape them using a backslash.

**Comment by bmarwell** [comment]
Sure, can do, BUT this would be completely different than what other plugins do  (e.g. surefire argline).

**Comment by aalmiray** [comment]
I'm confused now. Only one of these suggestions is the compatible with what other plugins do and that is replacing lone breaks with space/blanks?

**Comment by bmarwell** [comment]
I  personally would have expected that the plugin does that, yes: replacing line breaks with blanks. If there are reasons why this should not be done here, that is okay. I just wanted to report "surprising" behaviour here.

Haven't thought of escape sequences, though. First of all, this would make it incompatible with windows, as they use `^` for line repetition, whereas unix uses `\` in scripts. This would effectively result in `universal` being useless... probably not intended.
Second, no other plugin uses line continuation characters, at least none that I know of.

**Comment by aalmiray** [comment]
Good point on platform specific escaping. Then we simply replace line breaks for spaces.

**Comment by bmarwell** [comment]
This is probably a breaking change. Someone might have used `\` for their unix-only distributions.

**Comment by aalmiray** [comment]
Perhaps, but using `universal` with platform specific escaping format was a mistake in the first place. Better correct this sooner than later.

**Comment by aalmiray** [comment]
🎉 This issue has been resolved in `v1.14.0` ([Release Notes](https://github.com/jreleaser/jreleaser/releases/tag/v1.14.0))
